syntax/coq.vim

```diff
@@ -355,10 +355,15 @@ syn match coqField       contained "[_[:alpha:]][_'[:alnum:]]*"
 syn region coqCls        contains=coqClsProfile start="\<Class\>" matchgroup=coqVernacPunctuation end="\.\_s" keepend
 syn region coqClsProfile contained contains=coqIdent,coqRecTerm,coqRecBinder matchgroup=coqVernacCmd start="Class" matchgroup=NONE end="\.\_s"
 
+syn region coqCls matchgroup=coqVernacCmd start="\<Existing\_s\+Class\>" contains=coqIdent end="\.\_s"
+
 " Typeclass instances
-syn region coqIns contains=coqDefName matchgroup=coqVernacCmd start="\<\%(Declare\_s\+\)\?Instance\>" matchgroup=coqVernacPunctuation end=":="me=e-2 end="\.$"me=e-1 end="\.\_s"me=e-2 nextgroup=coqDefContents1,coqProofBody keepend skipnl skipwhite skipempty
+syn region coqIns contains=coqDefName matchgroup=coqVernacCmd start="\<\%(\%(Program\|Declare\)\_s\+\)\?Instance\>" matchgroup=coqVernacPunctuation end=":="me=e-2 end="\.$"me=e-1 end="\.\_s"me=e-2 nextgroup=coqDefContents1,coqProofBody keepend skipnl skipwhite skipempty
 syn region coqIns matchgroup=coqVernacCmd start="\<Existing\_s\+Instance\>" matchgroup=coqVernacPunctuation end="\.$"me=e-1 end="\.\s"me=e-2
 
+" Program
+syn region coqProgram matchgroup=coqVernacCmd start="\<Obligation\_s\+Tactic\_s\+:=\s\+" contains=coqIdent end="\.\_s"
```

Comment:
Could you fix the broken version of this region on line 123 (`syn region coqObligation`) instead?

Reply:
Sure. Done!